MAINTAINERS.md

```diff
@@ -17,76 +17,82 @@ The following is the answer to that question **for Astro Maintainers**, so that
+For PRs related to the Astro Docs website itself, including accessibility (a11y) issues, UI elements and project structure:
 
 - Test the proposed changes at a variety of screen sizes, and on a variety of devices.
 - Check both light and dark theme modes.
-- Navigate to several pages, including following anchor links, to test behaviour
+- Navigate to several pages, including following anchor links, to test behaviour.
 - Conduct any a11y checks or tests you have access to: screen reader behaviour, colour contrast ratios, etc.
 - For most non-trival PRs, the Docs Infrastructure Lead will likely want to do a final review themselves before the PR is merged.
 
 ### Language Translations
 
 For PRs that are translations to existing Docs content, including new page additions as well as smaller updates and corrections:
 
-- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the transators' process
+- Consult the [Internationalization Guide](/src/i18n/README.md) to familiarize yourself with the tranlsators' process.
 - If you see "LGTM" in a message within the PR, that means at least one other native speaker has approved the translation, and the PR can be immediately merged! 🥳
-- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`
-- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place.
+- If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`.
+- If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place. While we always prefer to have a review from a native speaker of the language, having translated docs with some errors is usually better than having no docs at all.
 
 
 ## Submitting your own PRs
 
 This section is much shorter now that we are protecting the `main` branch of the Astro repos! 😅
 
-Maintainers who submit PRs are expected to **merge their own PRs**, but only **after receiving an LGTM**. 
+If your PR depends on a core PR being merged first, please **mark your PR as a draft** so that it does not get merged first. Please also link to the other PR so that we can easily check on its status when we are doing PR reviews.
```

Comment:
```suggestion
If your PR depends on a core PR being merged first, please **mark your PR as a draft** so that it does not get merged first. Please also link to the other PR so that we can easily check on its status when we are doing PR reviews.

If your PR includes content for the Docs site to be translated and reflected in all languages, please **only submit your content in English**. This includes any related updates to the navigation sidebar etc. We have a robust system in place for updating the status of affected pages and all related site infrastructure in our other languages, and often our translators are in the best position to notice the "ripple effects" of your changes in their specific pages.

If your PR includes content changes that should *not* be reflected in other languages (e.g. an English typo fix), then you can add the word "minor" or "typo" in a commit message, and our system will *not* flag this page as needing a translation update. (This can be added in the "Squash and Merge" text field.)
```